src/patternfly/components/Table/examples/Table.md

```diff
@@ -2488,14 +2507,14 @@ Note: To apply padding to `.pf-c-table__expandable-row`, wrap the content in `.p
 {{/table}}
 ```
 
-### Usage
+### Hidden/visible breakpoint modifiers usage
 
 | Class | Applied to | Outcome |
 | -- | -- | -- |
 | `.pf-m-hidden{-on-[breakpoint]}` | `.pf-c-table tr > *` | Hides a table cell at a given breakpoint, or hides it at all breakpoints with `.pf-m-hidden`. **Note: Needs to apply to all cells in the column you want to hide.** |
 | `.pf-m-visible{-on-[breakpoint]}` | `.pf-c-table tr > *` | Shows a table cell at a given breakpoint. |
 
-## Controlling text
+## Controlling text modifiers
```

Comment:
Also `The table-text element` should probably be a subsection under `Controlling text modifiers`, especially since under the table text section is `Controlling text modifiers usage`. I wonder if we could just remove the `The table-text element` header and include the paragraph below it, and rename `Table text element` to `Controlling text using the table text element example`. wdyt?

Reply:
sounds good to me!